emukit/experimental_design/model_based/batch_experimental_design.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+from emukit.core.acquisition import Acquisition
+from emukit.core.interfaces import IModel
+from emukit.core.loop import CandidatePointCalculator, LoopState
+from emukit.core.optimization import AcquisitionOptimizer
+
+
+class GreedyBatchPointCalculator(CandidatePointCalculator):
+    """
+    Batch point calculator for use in batch experimental design. This point calculator calculates the first point in
+    the batch then adds this as a fake observation in the model with a Y value equal to the mean prediction. The model
+    is reset with the original data at the end of collecting a batch but if you use a model with a non-deterministic
```

Comment:
What is "non-deterministic training regime"?

Reply:
I believe non-deterministic training means that if you condition the same model on the same data but with different random seed, then the trained model will be different? Is that correct, Mark? Another source of difference might be numerical errors.

Reply:
Sorry, I should have made a point clearer. Please add an explanation of this phrase to the docstring, so that the question does not arise.

Reply:
regime -> procedure might be good enough.

Reply:
Rephrased